cmd/pitr/pxc/pxc.go

```diff
@@ -187,12 +208,80 @@ func GetPXCLastHost(pxcServiceName string) (string, error) {
 		}
 	}
 	if len(lastHost) == 0 {
-		return "", errors.New("cant find host")
+		return "", errors.New("can't find host")
 	}
 
 	return lastHost, nil
 }
 
+func GetPXCOldestBinlogHost(pxcServiceName, user, pass string) (string, error) {
+	cmd := exec.Command("peer-list", "-on-start=/usr/bin/get-pxc-state", "-service="+pxcServiceName)
```

Comment:
can we move this part to separate function?
We already using the same code in 2 places

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/761/commits/ea34dccb63d118b921baf514be8d5993d1a14184